src/lib/db/migrations/20230310111029_initial.ts

```diff
@@ -62,20 +65,23 @@ export async function up(knex: Knex): Promise<void> {
         useNative: true,
       })
       .notNullable()
+      .defaultTo('submitted')
     def.datetime('created_at').notNullable().defaultTo(now())
     def.datetime('updated_at').notNullable().defaultTo(now())
     def.integer('token_id')
     def.primary(['id'])
     def.specificType('hash', 'CHAR(64)').notNullable()
-    def.enu('api_type', ['example_a', 'example_b'], { useNative: true, enumName: 'api_type' })
-    def.enu('transaction_type', ['creation', 'proposal', 'accept'], { useNative: true, enumName: 'transaction_type' })
+    def.enu('api_type', ['certificate', 'attachment'], { useNative: true, enumName: 'api_type' }).notNullable()
```

Comment:
there will never be a transaction with an `attachment` type. I think we can remove from the enum, but keep the field as we know there will be more types in time

Reply:
understood